### PR TITLE
Add jenkins scripts

### PR DESCRIPTION
**Pull request by awisniew90**

### CODE REVIEW

File: ci/jenkins/release [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        // }
+                
+        stage('Export') {
+            steps {
+                dir('liberty-tools-eclipse/releng/io.openliberty.tools.update/target') {
+                    
+                    // Zip repository directory
+                    sh "cd repository && zip -r repository.zip * && mv repository.zip .."
+                    
+                    archiveArtifacts artifacts: 'repository/**/*.*', fingerprint: true
```
Comment (scottkurz) [inline ci/jenkins/release:153]: Maybe a dumb question but what's 'archiveArtifacts' ?  Is that a Jenkins command?

Comment (awisniew90) [reply ci/jenkins/release:153]: Yup - that "saves" the specified files for that build

File: ci/jenkins/release [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                        sh "cd features && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${FEATURE_JAR} -o ${FEATURE_JAR} --user '$user:$password'"
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${LS_JAR} -o ${LS_JAR} --user '$user:$password'"
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${UI_JAR} -o ${UI_JAR} --user '$user:$password'"
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${PRODUCT_JAR} -o ${PRODUCT_JAR} --user '$user:$password'"
+                    }  
+                }
+                
+                dir('liberty-tools-eclipse') {
+                    // Run Tycho plugin to recreate artifacts conten
+                    sh "mvn -pl releng tycho-p2-repository:fix-artifacts-metadata -Drepository.home=io.openliberty.tools.update"
```
Comment (scottkurz) [inline ci/jenkins/release:119]: I vaguely remember why we do this "fix" but could you please remind me again?

File: ci/jenkins/run-tests [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        
+        stage('Test on Eclipse 1Q2023') {
+            steps {
+                dir('liberty-tools-eclipse') {
+                    catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+                        script {
+                            try {
+                                wrap([$class: 'Xvfb', installationName: 'gui-test', autoDisplayName: true, debug: true, parallelBuild: true, screen: '1680x1050x24']) {
+                                    sh "metacity --sm-disable --replace &"
+                                    sh "mvn clean verify -Dtycho.disableP2Mirrors=true -Declipse.target=1Q2023 -Dosgi.debug=./tests/resources/ci/debug.opts -DmvnImportWait=60000 -Dtycho.showEclipseLog -e"
```
Comment (scottkurz) [inline ci/jenkins/run-tests:89]: Why do we do:  `-Dtycho.disableP2Mirrors=true`  ?  Did we run into trouble?

Comment (awisniew90) [reply ci/jenkins/run-tests:89]: Yea we hit issues without setting this.  I cant remember the exact details, but this didnt seem to impact performance or anything and got past the problem (whatever it was :-) )

**Comment by awisniew90** [comment]
Agreed - nothing seems "sensitive" to omit.

**Pull request merged**